Sources/Fullscreen/FrontpageSearch/Models/FrontpageSearchGroupItem.swift

```diff
@@ -12,6 +12,7 @@ public struct FrontpageSearchGroupItem: Hashable {
     public let displayType: FrontpageResultItemType
     public let adId: Int?
     public let isFavorite: Bool?
+    public let imageContentModeFill: Bool?
```

Comment:
Instead of having a boolean, it can be more explicit by using `UIView.ContentMode`, with `.scaleAspectFill` as the default.

Reply:
Yeah, I tried that but because UIView.ContentMode is not Hashable it started complaining, and instead of trying to make the code run despite including it I just went with this simpler solution. What do you think? Still worth changing?